mulighetsrommet-api/src/main/resources/db/migration/R__avtale_admin_view.sql

```diff
@@ -1,27 +1,35 @@
-create or replace view avtale_admin_dto_view as
+drop view if exists avtale_admin_dto_view;
+create view avtale_admin_dto_view as
 select a.id,
        a.navn,
        a.tiltakstype_id,
        a.avtalenummer,
        a.leverandor_organisasjonsnummer,
-       vk.id                  as leverandor_kontaktperson_id,
-       vk.organisasjonsnummer as leverandor_kontaktperson_organisasjonsnummer,
-       vk.navn                as leverandor_kontaktperson_navn,
-       vk.telefon             as leverandor_kontaktperson_telefon,
-       vk.epost               as leverandor_kontaktperson_epost,
-       vk.beskrivelse         as leverandor_kontaktperson_beskrivelse,
-       v.navn                 as leverandor_navn,
+       vk.id                                                                    as leverandor_kontaktperson_id,
+       vk.organisasjonsnummer                                                   as leverandor_kontaktperson_organisasjonsnummer,
+       vk.navn                                                                  as leverandor_kontaktperson_navn,
+       vk.telefon                                                               as leverandor_kontaktperson_telefon,
+       vk.epost                                                                 as leverandor_kontaktperson_epost,
+       vk.beskrivelse                                                           as leverandor_kontaktperson_beskrivelse,
+       v.navn                                                                   as leverandor_navn,
        a.start_dato,
        a.slutt_dato,
-       a.arena_ansvarlig_enhet,
+       case
+           when arena_ansvarlig_enhet is null then null::jsonb
+           else
+               jsonb_build_object(
+                       'enhetsnummer', arena_nav_enhet.enhetsnummer,
+                       'navn', arena_nav_enhet.navn,
```

Comment:
Vi burde dobbeltsjekke at alle eksisterende `arena_ansvarlig_enhet` faktisk finnes i db'en. Hvis det er slik at det kan komme gjennomføringer fra arena med en enhet vi ikke kjenner til så vil deserialiseringen av dette objektet krasje :thinking:

Reply:
Godt poeng, tror faktisk jeg så noen i min lokale database der `arena_ansvarlig_enhet` var satt, men der den ikke fant enheten. Jeg tar og sjekker.

Reply:
Ser disse to spørringene 
```sql
SELECT DISTINCT arena_ansvarlig_enhet
FROM avtale
WHERE NOT EXISTS (
    SELECT 1
    FROM nav_enhet
    WHERE nav_enhet.enhetsnummer = avtale.arena_ansvarlig_enhet
);
-- Gir to rader, en med null og en med enhetsnummer 9969

SELECT DISTINCT arena_ansvarlig_enhet
FROM tiltaksgjennomforing
WHERE NOT EXISTS (
    SELECT 1
    FROM nav_enhet
    WHERE nav_enhet.enhetsnummer = tiltaksgjennomforing.arena_ansvarlig_enhet
);

 Gir 137 rader med diverse enhetsnummer som ikke eksisterer...
```

Gir treff på noen enheter som ikke finnes lenger...

Reply:
Jeg har oppdatert PRen nå med å også sjekke at vi har verdier etter join i viewet for avtaler og tiltaksgjennomføringer, samt skrevet en test som sjekker at ikke alt eksploderer hvis vi av en eller annen grunn ikke har ansvarlig enhet i enheter-tabellen.